docs/content/integrations/social-authentication.md

```diff
@@ -20,7 +20,7 @@ leverage Auth0 to authenticate users on DefectDojo.
     -   Domain
     -   Client ID
     -   Client Secret
-4.  Now, edit the dojo/settings.py file and edit/replace the following
+4.  Now, edit the `dojo/settings/settings.py` file and edit/replace the following
```

Comment:
You should edit settings.dist.py or add the config in local_settings.py or use environment variables

Reply:
So you want to modify it by `dojo/settings/settings.dist.py` ?